### ISSUE TITLE
Allow looped samples to stretch based on project tempo

### ISSUE THREAD

**Issue by IMTG-Music** [LMMS/lmms#5474]
Loop users would love to see their samples automatically adapt to the project BPM. ~~The audio file processor uses a resample method which changes both BPM and pitch. A stretching option could allow BPM adaptation only.~~ #3103

**Comment by SecondFlight** [comment]
Time stretching is a duplicate of #3103. When you say sample BPM detection, do you mean like what's in FL, where the sample can be automatically stretched to fill a bar (or some fraction or multiple of a bar)?

**Comment by IMTG-Music** [comment]
Yes that's what I meant, sorry for the duplicate I didnt see it !
